nodeServer/ts/callService.ts

```diff
@@ -0,0 +1,66 @@
+import { Response, Request } from "express";
+import { Router } from "express-serve-static-core";
+
+
+
+/*
+  This program and the accompanying materials are
+  made available under the terms of the Eclipse Public License v2.0 which accompanies
+  this distribution, and is available at https://www.eclipse.org/legal/epl-v20.html
+  
+  SPDX-License-Identifier: EPL-2.0
+  
+  Copyright Contributors to the Zowe Project.
+*/
+
+const express = require('express');
+const Promise = require('bluebird');
+
+class CallService{
+  private context: any;
+  private router: Router;
+  
+  constructor(context: any){
+    this.context = context;
+    let router = express.Router();
+    router.use(function noteRequest(req: Request,res: Response,next: any) {
+      context.logger.info('Saw request, method='+req.method);
+      next();
+    });
+    context.addBodyParseMiddleware(router);
+    router.post('/',function(req: Request,res: Response) {
+      (req as any).zluxData.plugin.callService('hello', '', {method:'POST', body: req.body, contentType:"text/plain"}).then((callRes)=> {
```

Comment:
I am curious why do we need to pass `text/plain` unconditionally. What if we pass an original content-type as `req.body['content-type']` or omit it all all?

Reply:
Good point, its an oversight. I will fix it.